- added success handler, if admin or user redirect to /home;

dmitr2ish opened the pull request:
Added a handler for the success of the login, where we can customize the script successfully login to the system. The handler can determine which role the current user has. Now user and administrator are redirected to the same "/home" page. We can change the address if necessary.

dmitr2ish committed:
- added success handler, if admin or user redirect to /home;

kurmasasha commented on src/main/java/ru/javamentor/config/util/LoginSuccessHandler.java:
```diff
@@ -0,0 +1,43 @@
+                                        Authentication authentication) throws IOException {
+        User user = (User) authentication.getPrincipal();
+
+        boolean adminflag = false;
+        boolean userflag = false;
```
Название переменных по код-стайлу сделай

dmitr2ish replied:
Хорошо

dmitr2ish committed:
- added code style in Login succesHandler;
- added redirect admin to admin panel
- remove excess cast to user from principal;
